Client/Frontend/Components/FakespotErrorCardView.swift

```diff
@@ -0,0 +1,142 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Common
+
+final class FakespotErrorCardView: UIView, ThemeApplicable {
+    private enum UX {
+        static let buttonFontSize: CGFloat = 16
+        static let buttonVerticalInset: CGFloat = 12
+        static let buttonHorizontalInset: CGFloat = 16
+        static let buttonCornerRadius: CGFloat = 13
+        static let containerSpacing: CGFloat = 8
+        static let containerMargins = UIEdgeInsets(top: 8, left: 12, bottom: 8, right: 12)
+        static let iconStackViewSpacing: CGFloat = 4
+        static let horizontalStackViewSpacing: CGFloat = 12
+        static let verticalStackViewSpacing: CGFloat = 4
+        static let iconSize: CGFloat = 24
+        static let titleFontSize: CGFloat = 13
+        static let descriptionFontSize: CGFloat = 13
+        static let cornerRadius: CGFloat = 4
+    }
+
+    private lazy var containerStackView: UIStackView = .build { stackView in
+        stackView.addArrangedSubview(self.infoContainerStackView)
+        stackView.addArrangedSubview(UIView())
```

Comment:
What's the reason for adding an empty view here and in other stack views? Also adding views to the stack view should happen in `setupLayout()` so that the layout is all in one place and we don't have to use self.

Reply:
It's an old trick to add space and center things in UIStackViews. In SwiftUI it has a name, it's called 'Spacer'. It's a way of achieving the layout without many constraints. You probably noticed I have only a few for the image and the stack and it's a looot cleaner.

Reply:
As mentioned in another PR: we shouldn't use `self` in `.build`. Please move all the `addArrangedSubview` calls to `setupLayout()` as we would like to have how the layout it build in one place only.
In general we also avoid using so many instances of `UIStackView` as we noticed that they tend to cause issues with the layout.

Reply:
Okay, I will move the self code. As for multiple instances of `UIStackView` changing this will mean that I have to redo entirely UI and spend more time there. Can you confirm it's what we want to do?

Reply:
For this one we can leave it like this and rework it when we notice issues. I'd ask however to not use more than 1-2 stackviews for a component in the future.